### PR TITLE
Namespace plugin endpoints with Django app label

### PR DESCRIPTION
**Pull request by goosemania**
closes #4279
https://pulp.plan.io/issues/4279

### CODE REVIEW

**Comment by dralley** [comment]
@goosemania Approach looks good, LGTM

File: pulpcore/tests/functional/api/using_plugin/constants.py [hunk 1]
```diff
@@ -9,13 +9,14 @@
     CONTENT_PATH
 )
 
-FILE_CONTENT_NAME = 'pulp_file.file'
 
-FILE_CONTENT_PATH = urljoin(CONTENT_PATH, 'file/files/')
+FILE_CONTENT_NAME = 'file.file'
 
-FILE_REMOTE_PATH = urljoin(BASE_REMOTE_PATH, 'file/')
+FILE_CONTENT_PATH = urljoin(CONTENT_PATH, 'files/')
```
Comment (daviddavis) [inline pulpcore/tests/functional/api/using_plugin/constants.py]: I'm not sure I understand this change. Shouldn't `FILE_CONTENT_PATH` still be `/pulp/api/v3/content/file/files/` and not `/pulp/api/v3/content/files/`?

Comment (goosemania) [reply pulpcore/tests/functional/api/using_plugin/constants.py]: @daviddavis , you're right. Thank you!
I was surprised, why tests didn't fail, so after some investigation, I found a problem with installation of pulp_file, so test were skipped. Fixed as well now

**Comment by goosemania** [comment]
@daviddavis , ready for re-review

**Comment by bmbouter** [comment]
This looks good to me. Thanks for putting this together!

**Pull request merged**